plugins/omemo/src/ui/manage_key_dialog.vala

```diff
@@ -123,22 +123,22 @@ public class ManageKeyDialog : Gtk.Dialog {
         ListBoxRow verify_row = new ListBoxRow() { visible = true };
         verify_row.add(make_action_box(_("Verify key fingerprint"), _("Compare this key's fingerprint with the fingerprint displayed on the contact's device.")));
         ListBoxRow reject_row = new ListBoxRow() { visible = true };
-        reject_row.add(make_action_box(_("Reject key"), _("Stop accepting this key during communication with its associated contact.")));
```

Comment:
"Stop accepting" sounds like the software is accepting the key over and over when writing with this contact and should now stop it. That gives a confusing picture of what actually happens.

Also, a key relates to only one device of the contact.